src/main/java/com/github/piedpiper/node/stepfunctions/StepFunctionsExecute.java

```diff
@@ -0,0 +1,70 @@
+package com.github.piedpiper.node.stepfunctions;
+
+import java.util.Optional;
+
+import com.amazonaws.services.stepfunctions.AWSStepFunctions;
+import com.amazonaws.services.stepfunctions.model.StartExecutionRequest;
+import com.amazonaws.services.stepfunctions.model.StartExecutionResult;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.github.commons.utils.JsonUtils;
+import com.github.piedpiper.common.PiedPiperConstants;
+import com.github.piedpiper.node.INode;
+import com.github.piedpiper.node.NodeInput;
+import com.github.piedpiper.node.NodeOutput;
+import com.github.piedpiper.node.ParameterMetadata;
+import com.github.piedpiper.node.aws.IStepFunctionsFactory;
+import com.github.piedpiper.utils.ParameterUtils;
+import com.google.inject.Inject;
+
+public class StepFunctionsExecute implements INode {
```

Comment:
Do not extend INode unless u want it to come up in list of nodes. Extend Function itself rather